src/pollution_model/run_medusa_2.py

```diff
@@ -363,7 +393,7 @@ def dissolved_metal_load(total_copper_load: float, total_zinc_load: float,
     Returns
     -------
     MetalLoads
-        Returns the dissolved copper and zinc load for this surface
+        Returns the dissolved copper and zinc load for this surface (micrograms)
```

Comment:
The micrograms values don't match with the latest front-end changes, so I am blocking this PR against #273, once #273 is complete, you should ensure the units are correct in the WPS process descriptions